### PR TITLE
Added Dropdown to user to Header

### PR DESCRIPTION
**Pull request by magsout**
Need to populate link inside dropdown
When button is clicked, just added as usual `is-active` to `wc-DropdownHeader`
and for accessibility   change value of `aria-pressed`

r? @miketaylr

### CODE REVIEW

**Comment by magsout** [comment]
Ah hum and yes, I need to rewrite some test about logout user.

**Comment by miketaylr** [comment]
:balloon:

**Comment by magsout** [comment]
Just pushed my last commit for these beautiful and awesome days. 
Thanks again @miketaylr @karlcow @hallvors . Love u guys .

**Comment by karlcow** [comment]
aaaaw :heart: Thanks a lot for the amazing work.

**Comment by miketaylr** [comment]
:tada:

**Comment by miketaylr** [comment]
> Ah hum and yes, I need to rewrite some test about logout user.

I'm happy to re-write the logout test stuff, no worries.

**Comment by miketaylr** [comment]
Ah, actually I changed the test yesterday that instead of clicking on a link, it just navigates directly to the `/logout` route -- so no test bustage here.

File: webcompat/templates/topbar.html [hunk 1]
```diff
@@ -32,12 +32,17 @@
...
+                <span class="wc-Navbar-link-icon wc-Icon wc-Icon--chevron-down" aria-hidden="true"></span>
+              </button>
+              <div class="wc-DropdownHeader-content">
+                <a class="wc-Navbar-linkDropdown" href="{{ url_for('logout') }}">My activities</a>
+                <a class="wc-Navbar-linkDropdown" href="{{ url_for('logout') }}">Logout</a>
+              </div>
+            </div>
             {% if session.user_id and session.avatar_url %}
-            <img class="wc-Navbar-avatar" src="{{ session.avatar_url }}s=40" alt="User Avatar">
             {% endif %}
```
Comment (miketaylr) [inline webcompat/templates/topbar.html]: We can kill this empty `{% if session.user_id and session.avatar_url %}{%endif%}` block now, I think. :hocho:

**Comment by miketaylr** [comment]
And I guess we can make this change for nav.html as well.

**Comment by magsout** [comment]
ah yes, it's a different view. I forgot it. There is a way to mixe this two views ?

**Comment by miketaylr** [comment]
I was just wondering about that. Looking at them both, we can pull out the "All issues" "login" "logout" links into its own partial and at least share that code.

**Comment by miketaylr** [comment]
I just pulled out the common stuff and made a `shared-nav-links.html` partial, so we're sharing those in top.html and nav.html. Ideally we could just have the same file though, let me see if there's a way in Flask to know what page we're on and we can do some kind of if/else display block (or something).

**Comment by miketaylr** [comment]
@magsout I rebased against master (and force pushed) just now because I need to fix the login test, so please do a `git pull --rebase` or similar before trying to push on this branch. ^_^

**Comment by miketaylr** [comment]
@magsout just a thought, we could land this on master now if we commented out the "My Activity" link. Then we could work on the me-dashboard in a new branch. WDYT?

**Comment by magsout** [comment]
yeah, looks good to me.

**Comment by miketaylr** [comment]
Cool, I'll comment that out and then merge this.

**Comment by miketaylr** [comment]
r+ for @magsout work. I'll ask for review on the JS additions when we land the dashboard (or something).

**Comment by magsout** [comment]
awesome

**Comment by miketaylr** [comment]
ahahahah

**Comment by magsout** [comment]
Call me : Mr GIF

**Pull request merged**